OPHD/UI/FactoryProduction.cpp

```diff
@@ -218,26 +212,24 @@ void FactoryProduction::update()
 
 	Window::update();
 
-	auto& renderer = Utility<Renderer>::get();
-
-	const auto drawTitleText = [&renderer](NAS2D::Point<int> position, const std::string& title, const std::string& text) {
-		renderer.drawText(*FONT_BOLD, title, position, NAS2D::Color::White);
-		position.x() += 120;
-		renderer.drawText(*FONT, text, position, NAS2D::Color::White);
-	};
-
+	const int labelWidth = 120;
 	auto position = mRect.startPoint() + NAS2D::Vector{constants::MARGIN * 2 + static_cast<int>(mProductGrid.width()), 25};
-	drawTitleText(position, "Turns Completed:", std::to_string(mFactory->productionTurnsCompleted()) + " of " + std::to_string(mProductCost.turnsToBuild()));
+	auto value = std::to_string(mFactory->productionTurnsCompleted()) + " of " + std::to_string(mProductCost.turnsToBuild());
+	drawLabelAndValueLeftJustify(position, "Turns Completed:", value, 120);
 
 	position.y() += 20;
-	drawTitleText(position, "Common Metals:", std::to_string(mProductCost.commonMetals() * mProductCost.turnsToBuild()));
+	value = std::to_string(mProductCost.commonMetals() * mProductCost.turnsToBuild());
+	drawLabelAndValueLeftJustify(position, "Common Metals:", value, labelWidth);
```

Comment:
I'm not super fond of re-using the variable `value` for a bunch of different unrelated values.

I was thinking, if the parameter order of `drawLabelAndValueLeftJustify` was changed to move `labelWidth` right after `position`, it might make inline values more attractive:
```cpp
drawLabelAndValueLeftJustify(position, labelWidth, "Common Metals:", std::to_string(mProductCost.commonMetals() * mProductCost.turnsToBuild()));
```

Reply:
I like the idea of moving labelWidth right after position and will implement.

The purpose of reusing value was to reduce the length of the calls to drawLabelAndValue and (hopefully) make the code easier to read. Some of the lines were getting pretty long concatenating the strings within the function. We could change to using individual variables names instead of reusing value or just accept the line length will be long for these function calls in their current format. Certainly open to future refactoring of the RenderText functions to reduce their argument count.